roles/libvirt_manager/tasks/ocp_layout.yml

```diff
@@ -73,13 +74,30 @@
 
     - name: Create blank images for OCP cluster resources
       vars:
+        _matcher: >-
```

Comment:
if we could avoid injecting that again - afaik there's no use for it anymore.

Reply:
Removed